src/control.rs

```diff
@@ -43,14 +43,45 @@ impl ShouldColorize {
     pub fn from_env() -> Self {
         use std::io;
 
+        // Treat NO_COLOR as an alias for CLICOLOR_FORCE == 0
+        let clicolor_force = match (
+            ShouldColorize::normalize_env(env::var("NO_COLOR")),
+            ShouldColorize::normalize_env(env::var("CLICOLOR_FORCE")),
+        ) {
+            // Give priority to an explicitly set CLICOLOR_FORCE.
+            (_, Some(b)) => Some(b),
+
+            // Otherwise, NO_COLOR being set should act as CLICOLOR_FORCE == 0
+            (Some(_), None) => Some(false),
+
+            (None, None) => None,
+        };
+
         ShouldColorize {
             clicolor: ShouldColorize::normalize_env(env::var("CLICOLOR")),
-            clicolor_force: ShouldColorize::normalize_env(env::var("CLICOLOR_FORCE")),
+            clicolor_force: clicolor_force,
             ..ShouldColorize::default()
         }
     }
 
     pub fn should_colorize(&self) -> bool {
+        // Excluding the manual override, which takes priority over all
```

Comment:
this block would be great as part of a documentation comment of `ShouldColorize::from_env`